tests/test_flatten.py

```diff
@@ -31,3 +46,63 @@ def test_flatten_dict():
             "contextInfo_context2": "info2",
         }
     ]
+
+
+def test_flatten_pd():
+    flattened_pd = flatten(
+        PD_DATA, nested_columns=["contextInfo", "neighborhood"]
+    )
+    assert flattened_pd.equals(
+        DataFrame(
+            [
+                {
+                    "answer": 42,
+                    "contextInfo_context1": "info1",
+                    "contextInfo_context2": "info2",
+                }
+            ]
+        )
+    )
+
+
+def test_flatten_df_is_called():
+    # There is a bug on Pandas that makes apply fails when called from Series with the default MagicMock
```

Comment:
Can you add the link to the issue (bug report) as a comment too?

Reply:
Sure, I will. Also I have subcribed to the issue to get news about it.